tests/Models/TeamAssignmentTest.php

```diff
@@ -17,10 +21,46 @@ public function creating_a_team_assignment_creates_an_action_via_the_team_assign
     /** @test */
     public function creating_an_assignment_fires_a_notification_to_the_assignee()
```

Comment:
This name doesn't really express what the test does. Looks like it's been this way for a while but let's change it anyways.